includes/wikia/parser/templatetypes/TemplateTypesParser.class.php

```diff
@@ -48,16 +46,22 @@ public static function onFetchTemplateAndTitle( &$text, &$finalTitle ) {
 	 * @return bool
 	 */
 	public static function onGetTemplateDom( $title, $args, $frame, &$outputText ) {
-		global $wgEnableScrollboxTemplateParsing;
+		global $wgEnableScrollboxTemplateParsing, $wgEnableQuoteTemplateParsing;
 		wfProfileIn( __METHOD__ );
 
 		if ( self::shouldTemplateBeParsed() && !is_null( $args ) ) {
 			$type = self::getTemplateType( $title );
+			$templateArgs = TemplateArgsHelper::getTemplateArgs( $args, $frame );
 
 			if ( $type === AutomaticTemplateTypes::TEMPLATE_SCROLLBOX && $wgEnableScrollboxTemplateParsing ) {
-				$outputText = ScrollboxTemplate::getLongestElement(
-					TemplateArgsHelper::getTemplateArgs( $args, $frame )
-				);
+				$outputText = ScrollboxTemplate::getLongestElement( $templateArgs );
+			}
+
+			if ( $type === AutomaticTemplateTypes::TEMPLATE_QUOTE ||
+				$type === TemplateClassificationService::TEMPLATE_QUOTE &&
+				$wgEnableQuoteTemplateParsing
```

Comment:
This condition may not be doing what you expect it to be doing - please add parenthesis